### PR TITLE
Updated to refer to Jira

### PR DESCRIPTION
**Pull request by MadelineMurray**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,6 +7,13 @@
 
 Orion is a Apache 2.0 licensed, private transaction manager.
 
+## Issues 
+
+Orion issues are tracked in [Jira](https://pegasys1.atlassian.net/secure/Dashboard.jspa?selectPageId=10000) not GitHub. 
+See our [contribution guidelines](CONTRIBUTING.md) for more detail on searching and creating issues. 
+
+If you've commented on existing issue and have been waiting for a reply, feel free to ping us on [Gitter].
```
Comment (arash009) [inline README.md]: There were no issues. Should we just say to contact us on Gitter for any questions and keep it generic?

Comment (arash009) [reply README.md]: Plus Gitter link

Comment (MadelineMurray) [reply README.md]: Gitter link is there in the actual page - it's just at the bottom of the MD file.

Comment (MadelineMurray) [reply README.md]: I did specifically want to call out the case where they've commented on something and there's not been a reply.  I think it's OK?  We say in the Contributing docs to get us on Gitter for any questions.

**Pull request merged**